internal/logger/target/http/http.go

```diff
@@ -121,54 +124,62 @@ func acceptedResponseStatusCode(code int) bool {
 }
 
 func (h *Target) startHTTPLogger() {
+	ctx, cancelFn := context.WithCancel(context.Background())
```

Comment:
Doing it like this is leaky, meaning queued entries will be dropped and never sent. I will see if I can make it non-leaky.

Reply:
Have made changes as suggested.